collie/model/base/base_pipeline.py

```diff
@@ -815,23 +818,21 @@ def user_user_similarity(self, user_id: int) -> pd.Series:
         user_embeddings = self._get_user_embeddings()
         user_embeddings = user_embeddings / user_embeddings.norm(dim=1)[:, None]
 
-        sim_score_idxs_series = self._similarity(embeddings=user_embeddings, id=user_id)
+        sim_score_idxs_series = self._calculate_embedding_similarity(
```

Comment:
Should the line above with `user_embeddings / user_embeddings.norm(dim=1)[:, None]` go into the `_calculate_embedding_similarity` method?

Reply:
oof, good call!